src/controllers/PlayerController.tsx

```diff
@@ -43,5 +45,26 @@ export const usePlayerController: PlayerController = ({player}) => {
     player?.set({...setState});
   };
 
-  return {isPlaying, duration, onStop, onPlay, onRestart, setAttribute};
+  const setSample = async (files?: FileList) => {
+    if (files && Array.from(files).length > 0) {
+      const fileName = files[0].name;
+      await player
+        ?.load(URL.createObjectURL(files[0]))
+        .then(() => setSampleName(fileName));
+      player?.stop();
+      player?.start();
```

Comment:
is there a function to just restart the player?

Reply:
yup! player?.restart();